React/Extensions/ReactExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using ReactForte.Extensions;
+using System.Collections.Generic;
+using System;
+using JavaScriptEngineSwitcher.Extensions.MsDependencyInjection;
+using JavaScriptEngineSwitcher.V8;
+using React.AspNet;
+using Jering.Javascript.NodeJS;
+using React.Services;
+using Microsoft.Extensions.Options;
+using ReactForte.Application.Webpack;
+using Microsoft.AspNetCore.Http;
+
+namespace React.Extensions;
+
+public static class ReactExtensions
+{
+    public static void AddReact(
+        this IServiceCollection services, IConfiguration configuration,
+        string rootPath)
```

Comment:
please align vertical formatting for methods names and arguments (sometimes tailing ')' is in a new line, sometimes after last argument, sometimes there is one arg per line, sometimes more)